hyperliquid/info.py

```diff
@@ -168,6 +168,47 @@ def meta(self) -> Meta:
             }
         """
         return cast(Meta, self.post("/info", {"type": "meta"}))
+    
+    def metaAndAssetCtxs(self) -> MetaAndAssetCtxs:
+        """Retrieve exchange MetaAndAssetCtxs
+
+        POST /info
+
+        Returns:
+            [
+                {
+                    universe: [
+                        {
+                            name: str,
+                            szDecimals: int
+                        },
+                        ...
+                    ]
+                },
+            [
+                {
+                    "dayNtlVlm": str,
+                    "funding": str,
+                    "impactPxs":
+                        [
+                         str,
+                         str
+                            ],
+                    "markPx": str,
+                    "midPx": str,
+                    "openInterest": str,
+                    "oraclePx": str,
+                    "premium": str,
+                    "prevDayPx": str
+                },
+                ...
+            ]
+
+
+        """
+        return cast(MetaAndAssetCtxs, self.post("/info", {"type": "metaAndAssetCtxs"}))  
+        # return self.post("/info", {"type": "metaAndAssetCtxs"})
```

Comment:
nit: remove commented out code

Reply:
Does this code have ask price and bid price??